requests/sessions.py

```diff
@@ -39,10 +39,7 @@
 
 # Preferred clock, based on which one is more accurate on a given system.
 if sys.platform == 'win32':
```

Comment:
Since we're now Python 3.7+ I wonder if we can start using `time.perf_counter` everywhere? Might need to do more research into why Windows was given only perf_counter/clock instead of `time.time()`